nixos/modules/services/networking/xrdp.nix

```diff
@@ -90,89 +99,110 @@ in
         '';
       };
 
-      confDir = mkOption {
-        type = types.path;
-        default = confDir;
-        defaultText = literalMD "generated from configuration";
-        description = lib.mdDoc "The location of the config files for xrdp.";
```

Comment:
Why this option was removed?

Reply:
You can override the files in /etc anyway, it became irrelevant.

You can still do aditional patches in the default one.

Reply:
So please deprecate the option properly using `mkRemovedOptionModule`.

Reply:
I don't necessarily agree with putting all the files in `/etc` (see below).

Reply:
I added it back as a read only option, stuff should be altered with `extraConfDirCommands`